test/e2e/lib/components/secure-payment-component.js

```diff
@@ -2,11 +2,12 @@
  * External dependencies
  */
 import { By, promise, until } from 'selenium-webdriver';
-import config from 'config';
 
 /**
  * Internal dependencies
  */
+// eslint-disable-next-line wpcalypso/no-package-relative-imports
+import config from 'config';
```

Comment:
Great question. I changed it because I'm used to see that import in the internal dependencies block ([here is an example](https://github.com/Automattic/wp-calypso/blob/master/client/login/controller.js#L11)) and because eslint was complaining about it showing me a `wpcalypso/no-package-relative-import` warning. 

But now that I think about it, I'm not sure if the package we import in the client code is the same we import in the tests code.

Reply:
Unfortunately, I'm not sure. In the example above I can see that it uses `isEnabled` method, which does not exist in a `config` used here. 

I think that `client` import is handled by webpack setup, which is not used in e2e tests.

I'd prefer to revert this change back, but if eslint complaining about it, I guess it would be fine to left it as is.